apps/generic-issuance-client/src/main.tsx

```diff
@@ -18,9 +18,51 @@ const router = createHashRouter([
   { path: "/pipelines/:id", element: <Pipeline /> }
 ]);
 
+function loadInitalState(): Partial<GIContextState> {
+  let isAdminMode = undefined;
+
+  try {
+    const adminSerializedValue =
+      window.localStorage.getItem("setting-admin-mode");
+
+    if (!adminSerializedValue) {
+      throw new Error();
+    }
```

Comment:
I don't really get why we need to throw an error here?

Reply:
i'm just being extra careful to handle the various error conditions - e.g. something is saved but not of the right type, etc.